test/remote/gateways/remote_cyber_source_test.rb

```diff
@@ -286,4 +286,19 @@ def test_successful_validate_pinless_debit_card
     assert_equal 'Y', response.params["status"]
     assert_equal true,  response.success?
   end
+
+  def test_network_tokenization_authorize_and_capture
```

Comment:
Is there ever a network tokenized purchase, or is it always auth+capture? Do refund semantics change?